js/functional.js

```diff
@@ -0,0 +1,62 @@
+var functional = (function() {
+
+  var inheritanceType = document.getElementById("inheritance-type").value;
+
+  var init = function() {
+
+    var teamName = document.getElementById("team-name").value,
```

Comment:
these fields should go inside the constructor, not in it. here and in other places